ejerciciosPilas/src/actores/libroPrimaria/Charco.ts

```diff
@@ -1,7 +1,26 @@
-/// <reference path="../ActorAnimado.ts"/>
-class Charco extends ActorAnimado {
+/// <reference path = "../../../../bower_components/pilasweb/dist/pilasweb.d.ts"/>
+/// <reference path="../ActorColisionable.ts"/>
+/// <reference path="../../actores/libroPrimaria/Coty.ts"/>
+/// <reference path="../../comportamientos/SaltarAnimado.ts"/>
+/// <reference path="../../comportamientos/Hundir.ts"/>
+
+class Charco extends ActorColisionable {
+
     constructor() {
         super(0, 0, { grilla: 'actor.charco.png' });
         this.definirAnimacion("parado", [0], 6, true);
     }
+
+    afecta() {
+        return [Coty];
```

Comment:
Esto hace falta? Me da mucha paja trabajar con la clase...

Reply:
Resuelvo, hablo abajo.

Reply:
@51lv3str1 fijate que los actores tienen etiquetas, justamente pensadas para no hacer el instance of